app/views/admin/reports/create.turbo_stream.haml

```diff
@@ -2,4 +2,4 @@
   = button t(:go), "report__submit-btn", "submit", disabled: true
 = turbo_stream.update "report-table" do
   = render "admin/reports/loading"
-= turbo_stream.action(:scroll_into_view, "report-table")
+= turbo_stream.scroll_into_view("#report-table", behavior: "smooth")
```

Comment:
Thanks for pointing out the difference between TurboStream and TurboPower helpers. Allowing a CSS selector gives a lot more flexibility.

Reply:
By the way, Turbo streams also has `action_all` which accepts targets with a CSS selector but it still doesn't allow for passing on attributes. Turbo Power is building the stream tag directly, without these helpers to achieve that.